src/bf_functions.py

```diff
@@ -344,231 +291,184 @@ def frequency2fft_bin(self, frequencies=all):
         n_fpgas = len(self.get_fpgas())
-
-	bf_fft_bins = n_chans/(bf_be_per_fpga*n_fpgas)
-	return bf_fft_bins
+        bf_fft_bins = n_chans/(bf_be_per_fpga*n_fpgas)
+        return bf_fft_bins
 
     def get_fft_bin_bandwidth(self):
-	"""get bandwidth of single fft bin"""
-	n_chans = self.get_param('n_chans')
+        """get bandwidth of single fft bin"""
+        n_chans = self.get_param('n_chans')
         bandwidth = self.get_param('bandwidth')
-
-	fft_bin_bandwidth = bandwidth/n_chans
-	return fft_bin_bandwidth
+        fft_bin_bandwidth = bandwidth/n_chans
+        return fft_bin_bandwidth
 
     def fft_bin2frequency(self, fft_bins=all):
         """returns a list of centre frequencies associated with the fft bins supplied"""
-	frequencies = []
-	n_chans = self.get_param('n_chans')
-
-	if fft_bins == all:
-	    fft_bins = range(n_chans)
-
-	if type(fft_bins) == int:
-	    fft_bins = [fft_bins]
-
-	if max(fft_bins) > n_chans or min(fft_bins) < 0:
-            raise fbfException(1, 'fft_bins out of range 0 -> %d' %(n_chans-1), \
-                               'function %s, line no %s\n' %(__name__, inspect.currentframe().f_lineno), \
-                               self.syslogger)
-
+        frequencies = []
+        n_chans = self.get_param('n_chans')
+        if fft_bins == all:
+            fft_bins = range(n_chans)
+        if type(fft_bins) == int:
+            fft_bins = [fft_bins]
+        if max(fft_bins) > n_chans or min(fft_bins) < 0:
+            raise fbfException(1, 'fft_bins out of range 0 -> %d' %(n_chans-1),
+                               'function %s, line no %s\n' %(__name__, inspect.currentframe().f_lineno), self.syslogger)
         bandwidth = self.get_param('bandwidth')
-
-	for fft_bin in fft_bins:
-	    frequencies.append((float(fft_bin)/n_chans)*bandwidth)
-
-	return frequencies
+        for fft_bin in fft_bins:
+            frequencies.append((float(fft_bin)/n_chans)*bandwidth)
+        return frequencies
 
     def frequency2fpga_bf(self, frequencies=all, fft_bins=[], unique=False):
         """returns a list of dictionaries {fpga, beamformer_index} based on frequency. unique gives only unique values"""
         locations = []
-
         if unique != True and unique != False:
-            raise fbfException(1, 'unique must be True or False', \
-                               'function %s, line no %s\n' %(__name__, inspect.currentframe().f_lineno), \
-                               self.syslogger)
-
+            raise fbfException(1, 'unique must be True or False', 'function %s, line no %s\n' %
+                               (__name__, inspect.currentframe().f_lineno), self.syslogger)
         if len(fft_bins) == 0:
             fft_bins = self.frequency2fft_bin(frequencies)
-
-        fpgas = self.frequency2fpgas(fft_bins=fft_bins)
-        bfs = self.frequency2bf_label(fft_bins=fft_bins)
-
+            fpgas = self.frequency2fpgas(fft_bins=fft_bins)
+            bfs = self.frequency2bf_label(fft_bins=fft_bins)
```

Comment:
You've changed the meaning of the code here, since now it will only execute when the if statement above it is true. Is that intended?